### PR TITLE
Feature/polygons table

### PR DESCRIPTION
**Pull request by peetjvv**
creating table in database for polygons
added sort polygons table method

### CODE REVIEW

File: GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs [hunk 1]
```diff
@@ -402,5 +404,16 @@ public void SortCalendarDates()
...
             this.ExecuteNonQuery("ALTER TABLE calendar_date_sorted RENAME TO calendar_date");
         }
+
+        /// <summary>
+        /// Deletes and recreates the polygons table in a sorted order (first by poly_pt_seq then by id) - may take time
+        /// </summary>
+        public void SortPolygons()//TODO: test!
+        {
+            this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [polygons_sorted] ( [id] TEXT NOT NULL, [poly_pt_lat] REAL, [poly_pt_lon] REAL, [poly_pt_seq] INTEGER );");
+            this.ExecuteNonQuery("INSERT INTO polygons_sorted (id, poly_pt_lat, poly_pt_lon, poly_pt_seq) SELECT id, poly_pt_lat, poly_pt_lon, poly_pt_seq FROM polygons ORDER BY poly_pt_seq ASC, id ASC;");
```
Comment (timminata) [inline GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs]: Wait, this doesn't look right. Surely it should be sorted by ID, then poly_pt_seq? Unless I'm going crazy. Did you test this sql line? We probably won't really need sort to be fair though

File: GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs [hunk 2]
```diff
@@ -220,6 +220,8 @@ private void RebuildDB()
             this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [cleaned_stops] ( [stop_id] TEXT);");
             // CREATE TABLE TO STORE LOGS OF EDITS
             this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [log] ( [timestamp] TEXT, [action] TEXT, [route_id] TEXT, [pc_name] TEXT, [pc_ip] TEXT, [note] TEXT);");
+            // CREATE TABLE TO STORE POLYGONS
+            this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [polygons] ( [id] TEXT NOT NULL, [poly_pt_lat] REAL, [poly_pt_lon] REAL, [poly_pt_seq] INTEGER );");
```
Comment (timminata) [inline GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs:224]: I see we are creating the table here which is correct, do you not need to add other classed to access the info? Or how are you pulling the polygons into capture?

Comment (peetjvv) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs:224]: It's all in GTFSHelpers. It's very reliant on the GMapPolygon object in GMap. Didn't want to add the entire GMap.Net package to GTFS.

Comment (timminata) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs:224]: Yeah that's fine

File: GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs (hunk 1 above)
Comment (peetjvv) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs]: I didn't really test it. I only added it as a safeguard in case something went wrong. Will test it now.

Comment (timminata) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs]: Yeah, I think you just need to swap the order of the order bys else it will mix the ids

Comment (peetjvv) [reply GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs]: seems you were right, but now something else broke in GTFSHelpers. will fix that now.

**Pull request merged**